src/coreclr/vm/method.cpp

```diff
@@ -2011,10 +2011,9 @@ PCODE MethodDesc::TryGetMultiCallableAddrOfCode(CORINFO_ACCESS_FLAGS accessFlags
     }
     CONTRACTL_END
 
-    if (IsGenericMethodDefinition())
+    if (IsGenericMethodDefinition() || ContainsGenericVariables())
```

Comment:
nit:  `ContainsGenericVariables` includes a check for `IsGenericMethodDefinition`

Reply:
```suggestion
    if (ContainsGenericVariables())
```